idunn/utils/rate_limiter.py

```diff
@@ -10,14 +10,17 @@
 
 TooManyRequestsException = TooManyRequests
 
+
 @contextmanager
 def dummy_limit():
     yield
 
+
 class HTTPTooManyRequests(HTTPException):
     default_status_code = 429
     default_detail = 'Too Many Requests'
```

Comment:
this derived class may not work with FastAPI

Reply:
Is it tested somewhere?

Reply:
I will add a test for this, in a separate PR.

Reply:
A new test case is defined in https://github.com/QwantResearch/idunn/pull/102 to test this behavior

Reply:
Thanks!